nessie/models/development_db.py

```diff
@@ -139,35 +89,35 @@ def load_student_athletes():
         'team_code': 'TNW',
         'team_name': 'Women\'s Tennis',
     }
-    create_student(
+    _create_student(
```

Comment:
@johncrossman Is it possible that this ASC test data creation can go also? There's now fake-Redshift test data for students of all types in `fixtures/loch.sql`, so it might be that these are just creating duplicate rows and I missed removing them earlier.

Reply:
@johncrossman Sorry, ignore the above - I was thinking about new fixtures in the BOAC codebase. Too much overlap!